x-pack/docs/en/watcher/input/http.asciidoc

```diff
@@ -139,6 +139,28 @@ http://openweathermap.org/appid[OpenWeatherMap] service:
 --------------------------------------------------
 // NOTCONSOLE
 
+===== Using Token-Based Authentication
```

Comment:
```suggestion
===== Using token-based authentication
```

Reply:
Thanks @abdonpijpelink,  the minor changes you suggested are fine for me. For the rebase sure as it's my first time doing it  ^^